### PR TITLE
cleanup ns client

### PR DESCRIPTION
**Pull request by anwar6953**
Clean up namespace client.

resolves: https://issues.cask.co/browse/CDAP-1136
http://builds.cask.co/browse/CDAP-DUT853-2

### CODE REVIEW

File: cdap-client/src/main/java/co/cask/cdap/client/NamespaceClient.java [hunk 1]
```diff
@@ -117,23 +103,17 @@ public NamespaceMeta get(String namespaceId) throws IOException, UnAuthorizedAcc
...
-                                                 HttpURLConnection.HTTP_FORBIDDEN);
-      if (HttpURLConnection.HTTP_NOT_FOUND == response.getResponseCode()) {
-        throw new NotFoundException(NAMESPACE_ENTITY_TYPE, namespaceId);
-      }
-      if (HttpURLConnection.HTTP_FORBIDDEN == response.getResponseCode()) {
-        throw new CannotBeDeletedException(NAMESPACE_ENTITY_TYPE, namespaceId);
-      }
-    } finally {
-      config.setApiVersion(origVersion);
+    config.setApiVersion(Constants.Gateway.API_VERSION_3_TOKEN);
```
Comment (bdmogal) [inline cdap-client/src/main/java/co/cask/cdap/client/NamespaceClient.java]: Why is this needed only for delete?

Comment (anwar6953) [inline cdap-client/src/main/java/co/cask/cdap/client/NamespaceClient.java]: I missed removing this.

**Comment by bdmogal** [comment]
Thanks for doing this. Just one question, then LGTM.

**Comment by bdmogal** [comment]
:+1:

File: cdap-client/src/main/java/co/cask/cdap/client/NamespaceClient.java (hunk 1 above)
Comment (bdmogal) [inline cdap-client/src/main/java/co/cask/cdap/client/NamespaceClient.java]: So good catch to me? :)

Comment (anwar6953) [inline cdap-client/src/main/java/co/cask/cdap/client/NamespaceClient.java]: Yes. good catch to you! :100:
